test.md

```diff
@@ -186,6 +186,27 @@ only need to do it on the first one. If you have a `<th>`, set it there.
   </tr>
 </table>
 
+## Box
+
+Here is the sample code for a single-cell table masquerading as a box. There are other ways to create a box (*e.g.*, with an HTML "box" element) but this works for our purposes.
+
+1. Lorem ipsum dolor sit amet, wisi viverra voluptas cras, vestibulum ligula molestie sagittis quis, nulla elementum id. Scelerisque tempus at consectetuer, neque nascetur erat pharetra rhoncus.
```

Comment:
Reformat your list items to have close to 80-character column length. Put a second space between the . and the first word, so that you can align subsequent lines of your list:

```
1.  List item 1
    Still in list item 1
2.  List item 2
```